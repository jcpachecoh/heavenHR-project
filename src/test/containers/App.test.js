import React from 'react';
import App from '../../containers/App';
import * as TestUtils from 'react-addons-test-utils';
import {combineReducers, createStore} from 'redux';
import { Provider } from 'react-redux';
import * as reducers from '../../reducers';

const reducer = combineReducers(reducers);
const store = createStore(reducer);

describe('Testing <App /> component', () => {
    it('Should render App component ', () => {
    
        expect(TestUtils.createRenderer().render(
            <Provider store={store}>
                <App />
            </Provider>
          )).toMatchSnapshot()
    });
});
