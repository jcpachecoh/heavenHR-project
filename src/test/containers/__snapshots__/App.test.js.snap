// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <App /> component Should render App component  1`] = `<App />`;
